templates/index.html

```diff
@@ -1,56 +1,283 @@
       </div>
-    </main>
-
-    <!-- Add footer below -->
-
-  </body>
-</html>
+    </div>
+    <div class="col-6 u-equal-height">
+      <div class="p-card">
+        <h3 class="p-card__title">Production stack</h3>
+        <hr />
+        <p>For production environments, where stability, availability and scalability are the key drivers, Canonical provides a production&dash;grade stack consisting of:</p>
+        <ul class="p-list">
+          <li class="p-list__item is-ticked"><a href="https://maas.io/" class="p-link--external">MAAS</a> &ndash; for bare-metal provisioning.</li>
+          <li class="p-list__item is-ticked"><a href="https://jaas.ai/kubernetes" class="p-link--external">Charmed Kubernetes</a> &ndash;   highly available and scalable Kubernetes cluster for Charmed OSM hosting purposes.</li>
+          <li class="p-list__item is-ticked"><a href="https://jaas.ai/openstack" class="p-link--external">Charmed OpenStack</a> &ndash; a highly available and scalable OpenStack cluster that will be used as the VIM.
+          </li>
+          <li class="p-list__item is-ticked"><a href="https://jaas.ai/osm-ha/bundle" class="p-link--external">Charmed OSM HA bundle</a> &ndash; a highly available and scalable OSM cluster.</li>
+        </ul>
+        <p><a href="https://ubuntu.com/telecommunications" class="p-link--external">Learn more about Canonical’s solutions for telcos</a></p>
+      </div>
+    </div>
+  </div>
+</section>
+<section class="p-strip--accent is-shallow">
+  <div class="u-fixed-width">
+    <h2>Interested in trying Charmed OSM?</h2>
+    <p class="p-heading--four">Tell us your NFV story and talk to a Canonical telco expert.</p>
+    <p><a href="/contact-us" class="p-button--positive">Get in touch</a></p>
+    <p><a href="https://tutorials.ubuntu.com/tutorial/charmed-osm-get-started#0" class="p-link--inverted">Try it yourself&nbsp;&rsaquo;</a></p>
+  </div>
+</section>
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Learn more</h2>
+    <p>Canonical is a key contributor to the upstream OSM project, and an owner of the N2VC (Network Service to VNF Communication) component.</p>
+    <ul class="p-list">
+      <li class="p-list__item"><a href="https://www.youtube.com/watch?v=Sv3Z_gbRVgQ" class="p-link--external">Listen to Nathan Rader &ndash; Canonical&rsquo;s NFV Strategy Director &ndash; talking about OSM at the SDN NFV World Congress 2017</a></li>
+      <li class="p-list__item"><a href="https://www.youtube.com/watch?v=xxt0zmE1thQ&t=1s" class="p-link--external">Watch the presentation about Canonical&rsquo;s Development Stack for OSM</a></li>
+    </ul>
+  </div>
+</section>
+<section class="p-strip is-bordered">
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h3>Need a hand with OSM?</h3>
+      <p>Don&rsquo;t wait anymore. Roll-out your NFV platform together with Canonical.</p>
+      <a href="/contact-us" class="p-button--positive">Contact us</a>
```

Comment:
```suggestion
      <a href="/contact-us" class="p-button--neutral">Contact us</a>
```

Two primary buttons going to the same place is one thing, but four primary buttons going to three places is a bit much.